alicloud/resource_alicloud_db_instance.go

```diff
@@ -790,6 +879,15 @@ func resourceAlicloudDBInstanceRead(d *schema.ResourceData, meta interface{}) er
 	}
 	d.Set("ssl_status", sslAction["RequireUpdate"])
 	d.Set("ssl_action", d.Get("ssl_action"))
+	d.Set("client_ca_enabled", d.Get("client_ca_enabled"))
+	d.Set("client_crl_enabled", d.Get("client_crl_enabled"))
+	d.Set("ca_type", sslAction["CAType"])
+	d.Set("server_cert", sslAction["ServerCert"])
+	d.Set("server_key", sslAction["ServerKey"])
+	d.Set("client_ca_cert", sslAction["ClientCACert"])
+	d.Set("client_cert_revocation_list", sslAction["ClientCertRevocationList"])
+	d.Set("acl", sslAction["ACL"])
+	d.Set("replication_acl", sslAction["ReplicationACL"])
 
 	tdeInfo, err := rdsService.DescribeRdsTDEInfo(d.Id())
 	if err != nil && !IsExpectedErrors(err, []string{"InvaildEngineInRegion.ValueNotSupported", "InstanceEngineType.NotSupport", "OperationDenied.DBInstanceType"}) {
```

Comment:
Please add docs for above the attaibutes.

Reply:
ok, i'll add docs